flag "Info Micromega" to print used hypotheses (#19703)
coq/coq: doc/sphinx/addendum/micromega.rst

  generates a *proof cache* which makes it possible to rerun scripts
  even without CSDP.

.. flag:: Info Micromega

>    This :term:`flag` (unset by default) instructs the tactics :tacn:`lia`,
>    :tacn:`nia`, :tacn:`lra`, :tacn:`nra` and :tacn:`psatz` to print the
>    list of hypotheses needed by the proof.

Reviewer: ```suggestion
   Instructs the tactics :tacn:`lia`,
   :tacn:`nia`, :tacn:`lra`, :tacn:`nra` and :tacn:`psatz` to print the
   list of hypotheses needed by the proof.  The default is unset.
```

The `.. flag:: Info Micromega` visibly labels this as flag, so no need to repeat that a second time in words.
